### DISCUSSION TITLE
Changing MQTT_STATUS_ON/OFF

### DISCUSSION THREAD

**Discussion by nmctiglao** [arendst/Tasmota#12388] [Q&A] [answered]
Changing the values in my_user_config.h below does not seem to have any effect. It still uses "ON"/"OFF". How do I change it?

#define MQTT_STATUS_OFF        "0"             // [StateText1] Command or Status result when turned off (needs to be a string like "0" or "Off")
#define MQTT_STATUS_ON         "1"              // [StateText2] Command or Status result when turned on (needs to be a string like "1" or "On")

**Comment by barbudor** [comment, marked as answer]
Like many options, these hard-coded values are used on 1st time init of the settings when tasmota is run for the 1st time on a freshly flashed device.
If you upgrade a device where tasmota has already ran, the settings have already been initialized with the values in the 1st ran tasmota binary.

To validate your changes in your newly build fw, you must erase and reflash your device.
Or alternatively use 'reset 1' to clear the settings area and simulate a 1st run.
